packages/cli/src/setup/setup.mts

```diff
@@ -81,7 +81,7 @@ export const setup = async (autoSetup: boolean) => {
 
 	const config = await getConfigDiff(options)
 
-	await writeConfigToFile(config)
+	await writeConfigToFile(config, '.typesafe-i18n.json')
```

Comment:
don't you need to use `options.project` here too?

Reply:
When I wrote this, I think that this `setup` method is used only for `in-place` config generation, not for `generate config in path passed by --project path`

We can add path here too if needed

Reply:
I think it would be more consistent if someone is able to pass the -p flag also to the setup function.